tests/test_tokenize_integers.py

```diff
@@ -0,0 +1,28 @@
+"""All Unit tests which focus on arithmatic operations in the calculator.
+"""
+import pytest
+
+import calc
+
+
+def test_parse_single_digits():
+    input_text = "1+1"
+    calculator = calc.Calc(text=input_text)
+    assert calculator.parse() == 2
+
+def test_parse_multiple_digits():
+    input_text = "100+1"
+    calculator = calc.Calc(text=input_text)
+    assert calculator.parse() == 101
+
+def test_next_token_read_full_int():
+    # This test was added to fix a bug where the Token after an integer wasn't
+    # being correctly grabbed.
+
+    # NOTE: Potentially we don't need to add this to the assignment unit tests?
```

Comment:
Hmm... :thinking: Do you think we really need it? If you do then I'm OK with keeping it.